openapi/sms/v1/instructions.md

```diff
@@ -0,0 +1,244 @@
+# Instructions
+
+SAP Emarsys empowers marketers around the world to create truly personalized omnichannel experiences that deliver business outcomes. The Emarsys SMS Partner API allows partners to integrate into the Emarsys SMS channel. The partner API is REST based.
+
+## API Specification
+
+The API specification has been created using OpenAPI 3.0.3. It is available to be exported using the links below:
+
+- [Partner Service Specification](../partner-service.yaml)
+- [Partner Callbacks Specification](../partner-callbacks.yaml)
+
+The current functionality supported is:
+
+- [Self-service on-boarding for the client end user.](#client-onboarding).
+- [Sending SMS messages triggered by the Emarsys platform.](#outbound-messages)
+- [Recording delivery reports of sent messages.](#outbound-messages)
+- [Ingesting inbound SMS messages.](#inbound-messages)
+
+## Glossary
+
+---
+
+## Setup
+
+### Partner Onboarding
+
+To onboard we require:
+
+- Display name of the partner
+- Base URL for the partner service
+- Supported countries (ideally a list of ISO-3 country codes)
+- Custom Fields for the self-service customer on-boarding
```

Comment:
Maybe we could clarify here what these custom fields mean. E.g. reference the endpoint using it.

Reply:
good idea, i will add